carriers/dhl.js

```diff
@@ -0,0 +1,127 @@
+const async = require('async');
+const request = require('request');
+const geography = require('../util/geography');
+const moment = require('moment-timezone');
+const checkDigit = require('../util/checkDigit');
+
+function DHL(options) {
+    this.isTrackingNumberValid = function(trackingNumber) {
+        // remove whitespace
+        trackingNumber = trackingNumber.replace(/\s/g, '');
+        trackingNumber = trackingNumber.toUpperCase();
```

Comment:
Looking at the valid tracking numbers in the test file, none have alpha characters. We probably don't need this line.

Reply:
Whoops, looking at the tests we _are_ sending in some alpha characters in the invalidTrackingNumber list.
Carry on :)